app/src/main/java/com/example/habitsmasher/MainActivity.java

```diff
@@ -91,9 +94,32 @@ public boolean onOptionsItemSelected(MenuItem item) {
      * This helper method is responsible for opening the FollowUserDialog box
      */
     private void openFollowUserDialog() {
-        FollowUserDialog followUserDialog = new FollowUserDialog();
+        FollowUserDialog followUserDialog = new FollowUserDialog(new FollowUserDialogFragmentDismissHandler(),
+                                                                 getApplicationContext());
```

Comment:
The diff is a bit messed up, but I'm just passing in the new **handler** and the current context to the dialog